steps/mentor-calibre-drc/configure.yml

```diff
@@ -38,6 +42,7 @@ parameters:
   design_name: undefined
   # Use the rule deck "inputs/adk/${drc_rule_deck}"
   drc_rule_deck: calibre-drc-block.rule
+  full_chip: False
```

Comment:
This parameter won't be needed either if you do what I said above.

Basically, the ADK should have different versions of the DRC rule deck for each purpose:

- calibre-drc-block.rule -- block checks
- calibre-drc-chip.rule -- full chip checks
- calibre-drc-antenna.rule -- antenna checks
- etc.

Reply:
I think this is still needed in order to add the "_filled" suffix to the design name, which is comes from the fill step. The other alternative is to have the user change the design_name for the step in the configure.py, but that doesn't seem great.

Reply:
I think we really should just keep it in the configure.py. It feels odd to add a new parameter and a new command to change another parameter.

We would have to do it for just that step though, after the graph-wide parameter update:

```
>>> g.update_params( parameters )
>>> drc_fill.set_param( 'design_name', 'design_foo' )
```